docs/OnTrack/UI Enhancement/Component Reviews/grade-icon.md

```diff
@@ -0,0 +1,61 @@
+# Ontrack Component review
+
+## Team Member Name
+Shounak Bhalerao | S224743249
+
+## Component Name
+
+- `grade-icon.coffee`
+- `grade-icon.scss`
+- `grade-icon.tpl.html`
+
+## Component purpose
+
+The purpose of `grade-icon` is to provide a unified simple component, for displaying the grade student is expecting.
+This item, can only be seen on screens with Tutor or higher permissions.
```

Comment:
I don't think only tutors can see it. A student can as well through screens like the portfolio submission screen. Please make sure to identify all uses of grade-icon in the code to be safe. You can search 'grade-icon' in VS Code and see every mention of it in the repo.